src/pages/components/accordion/code.mdx

```diff
@@ -1,36 +1,201 @@
+description:
+  The accordion component delivers large amounts of content in a small space
+  through progressive disclosure. The user gets key details about the underlying
+  content and can choose to expand that content.
 tabs: ["Usage", "Style", "Code", "Accessibility"]
 ---
 
-<ComponentDemo
-  components={[
-    {
-      id: "accordion",
-      label: "Accordion",
-    },
-  ]}
->
-  <ComponentVariant
-    id="accordion"
-    knobs={{ Accordion: ["align"], AccordionItem: ["open"] }}
-    links={{
-      React:
-        "http://react.carbondesignsystem.com/?path=/story/accordion--default",
-      Angular:
-        "https://angular.carbondesignsystem.com/?path=/story/components-accordion--basic",
-      Vue:
-        "http://vue.carbondesignsystem.com/?path=/story/components-cvaccordion--default",
-      Vanilla: "https://the-carbon-components.netlify.com/?nav=accordion",
-    }}
-  >
-    {`
-  <Accordion>
-    <AccordionItem title="Title 1"><p>The accordion component delivers large amounts of content in a small space through progressive disclosure. The user gets key details about the underlying content and can choose to expand that content within the constraints of the accordion. Accordions work especially well on mobile interfaces or whenever vertical space is at a premium.</p></AccordionItem>
-    <AccordionItem title="Title 2"><p>The accordion component delivers large amounts of content in a small space through progressive disclosure. The user gets key details about the underlying content and can choose to expand that content within the constraints of the accordion. Accordions work especially well on mobile interfaces or whenever vertical space is at a premium.</p></AccordionItem>
-    <AccordionItem title="Title 3"><p>The accordion component delivers large amounts of content in a small space through progressive disclosure. The user gets key details about the underlying content and can choose to expand that content within the constraints of the accordion. Accordions work especially well on mobile interfaces or whenever vertical space is at a premium.</p></AccordionItem>
-  </Accordion>
-  `}
-  </ComponentVariant>
-</ComponentDemo>
+<PageDescription>
+
+How to build an accordion using React.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Code usage</AnchorLink>
+<AnchorLink>Component API</AnchorLink>
+<AnchorLink>Reference</AnchorLink>
+
+</AnchorLinks>
+
+## Code usage
+
+You can build an accordion using a combination of the `Accordion` and
+`AccordionItem` components. The `Accordion` components accepts a list of
+`AccordionItem` components as `children` and each `AccordionItem` is responsible
+for displaying the accordion's heading and panel content.
+
+You can configure the accordion item's heading using the `title` prop.
+Everything you pass in as a child of `AccordionItem` will be rendered in the
+accordion's panel.
+
+```jsx
+import { Accordion, AccordionItem } from "carbon-components-react";
```

Comment:
@vpicone any tips on getting prettier to format these with single quotes? 👀

Reply:
Sure, what for?

Reply:
Just for this one snippet?

Reply:
For codeblocks in `mdx` files, prettier will format them using the default configuration for the language. The default configuration for prettier in JSX files uses double quotes and we would like to use single quotes.